slang-checks/src/main/java/org/sonarsource/slang/checks/TooManyParametersCheck.java

```diff
@@ -62,4 +63,9 @@ public void initialize(InitContext init) {
     });
   }
 
+  private static boolean isOverrideMethod(FunctionDeclarationTree tree) {
+    return tree.modifiers().stream().anyMatch(mod ->
+      (mod instanceof ModifierTree) && ((ModifierTree) mod).kind() == ModifierTree.Kind.OVERRIDE);
```

Comment:
These conditions are always true in the current test cases. We should add more test cases.